index.js

```diff
@@ -49,6 +50,8 @@ module.exports = function (options) {
 
   options.credentials = !!options.credentials;
 
+  options.headersKeptOnError = options.headersKeptOnError === undefined || !!options.headersKeptOnError;
```

Comment:
Oops, this was from when I was planning on making it an array so you could only keep some. Will be fixed in just a bit.